ansible/roles/util-cfg-service/templates/apps.properties.j2

```diff
@@ -27,6 +27,7 @@ apps.workspace.default-app-categories       = {{ apps.user_subs }}
 apps.workspace.dev-app-category-index       = 0
 apps.workspace.favorites-app-category-index = 1
 apps.workspace.public-id                    = 00000000-0000-0000-0000-000000000000
+apps.workspace.metadata.category.attrs      = rdf:type, http://edamontology.org/has_topic
```

Comment:
It sounds like giving the flexibility to the admins to choose the attributes might take a lot of changes.  I wonder, since we're already adding the attributes to the configs here, if it would make sense to have the IRI-to-attribute mapping in the configs as well somehow?  The UI in that case could get rid of the hard-coded IRI-to-attribute mapping and could be updated to understand the mapping provided in the configs.  (Either way, a design discussion would probably be good)